pdu/Src/tempSensor.c

```diff
@@ -22,15 +24,28 @@
 /*********************************************************************************************************************/
 /*------------------------------------------------------Macros-------------------------------------------------------*/
 /*********************************************************************************************************************/
-
+#define SENSOR_ADDRESS_READ 0x91
+#define SENSOR_ADDRESS_WRITE 0x90
+#define CONFIG_REGISTER 0x01
+#define TEMP_REGISTER 0x00
 /*********************************************************************************************************************/
 /*-------------------------------------------------Global variables--------------------------------------------------*/
 /*********************************************************************************************************************/
-
+uint8_t tempregister_info [2];
```

Comment:
For slightly better readability, use a macro for the size of the temperature register.

Reply:
do temp_register_info.